pkg/controllers/v1alpha1/mpi_job_controller.go

```diff
@@ -1261,3 +1261,16 @@ func getLabelsMap(mpiJob *kubeflow.MPIJob) map[string]string {
 func getLauncherName(mpiJob *kubeflow.MPIJob) string {
 	return mpiJob.Name + launcherSuffix
 }
+
+func IsJobFinished(launcher *batchv1.Job) bool {
+    if launcher == nil {
```

Comment:
Thanks for your fix:) We have the judge `launcher != nil &&` before, So don't need this line.

Reply:
done